component/sections/hero/index.js

```diff
@@ -1,10 +1,18 @@
-import React from "react"
-import dynamic from "next/dynamic"
+import React from "react";
+import dynamic from "next/dynamic";
+
+const Variants = [
```

Comment:
`Variants` is defined as a constant array, please change to object.
With current implementation, adding this section on demo page shows no preview. Please see this [loom video](https://www.loom.com/share/69c7529ff0314f24a249dd24edda3a5f), thanks!

Reply:
Thanks! Definitely needs to be fixed.

Reply:
Fix in [`8090572` (#18)](https://github.com/webriq-pagebuilder/site-template-default/pull/18/commits/809057252ba29b2728a584b373b8cc1fa35b8486)

Reply:
Approved. Confirming fix.